build.gradle.kts

```diff
@@ -23,23 +24,16 @@ plugins.withType<JavaPlugin>().configureEach {
 group = "org.junit-pioneer"
 description = "JUnit 5 Extension Pack"
 
-val modularBuild : String by project
-val experimentalJavaVersion : String? by project
+val modularBuild: String by project
+val experimentalJavaVersion: String? by project
```

Comment:
in the future we could use this to actually provide our targeted version - and utilize toolchains fully - we have to set the --release option properly